wled00/FX.h

```diff
@@ -883,7 +882,7 @@ class WS2812FX {  // 96 bytes
     inline void trigger()                                     { _triggered = true; }  // Forces the next frame to be computed on all active segments.
     inline void setShowCallback(show_callback cb)             { _callback = cb; }
     inline void setTransition(uint16_t t)                     { _transitionDur = t; } // sets transition time (in ms)
-    inline void appendSegment(const Segment &seg = Segment()) { if (_segments.size() < getMaxSegments()) _segments.push_back(seg); }
+    inline void appendSegment(Segment &&seg = Segment())      { if (_segments.size() < getMaxSegments()) _segments.push_back(std::move(seg)); }
```

Comment:
I've replace this with emplace_back, does the same.

Reply:
Not sure how this could be replaced with emplace here?

Reply:
That is, if you want to keep this `appendSegment` function. Removing it opens options.

Reply:
Change its parameters to accept segment coordinates instead.